### ISSUE TITLE
Add visual indicator on hover to our internal links in menus

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#4561]
We currently have no hover states on our menus on designsystemet.no. We should have discussion on this. Might be related to WCAG: Use of color
The most common approach is to add an underline on hover. 

This is the menu: 
<img width="981" height="673" alt="Image" src="https://github.com/user-attachments/assets/2b3b0c5c-d46b-47af-94c4-60cbfccbb07e" />

**Comment by mrosvik** [comment]
We decided to add a hover style like the active state. See AvatarStack here:

<img width="279" height="313" alt="Image" src="https://github.com/user-attachments/assets/80c56a40-c0c4-4a35-856a-a1bba7261c56" />
